build.rs

```diff
@@ -655,7 +681,23 @@ fn nasm(file: &Path, arch: &str, out_file: &Path, include_dir: &Path) -> Command
         std::path::MAIN_SEPARATOR,
     )));
 
-    let mut c = Command::new("./target/tools/windows/nasm/nasm");
+    match arch {
+        "aarch64" => clang(out_file, include_dir, file),
+        _ => nasm(out_file, oformat, include_dir, file)
+    }
+}
+
+fn find_nasm() -> Option<PathBuf> {
```

Comment:
Especially since this PR doesn't actually use NASM, we don't need to change the way NASM is found in this PR. If we want to be more flexible with NASM then let's discuss that in another issue/PR.